// Imported from
// //depot/google3/google/internal/chrome/cros/edu/schooltools/v1/bundle.proto

syntax = "proto3";

package boca;

message Bundle {
  repeated ContentConfig content_configs = 1;

  bool locked = 2;
}

message ContentConfig {
  oneof content {
    string url = 1;

    ClassroomContent classroom_content = 2;
  }
  string title = 3;

  string favicon_url = 4;

  LockedNavigationOptions locked_navigation_options = 5;
}

message ClassroomContent {
  string course_id = 1;

  string course_work_id = 2;
}

message LockedNavigationOptions {
  enum NavigationType {
    NAVIGATION_TYPE_UNKNOWN = 0;

    OPEN_NAVIGATION = 1;

    BLOCK_NAVIGATION = 2;

    DOMAIN_NAVIGATION = 3;

    LIMITED_NAVIGATION = 4;
  }

  NavigationType navigation_type = 1;
}
